zips/zip-0230.rst

```diff
@@ -436,7 +436,7 @@ An issuance action, ``IssueAction``, is the instance of issuing a specific Custo
 +-----------------------------+------------------------------+------------------------------------------------+---------------------------------------------------------------------+
 | varies                      |``nNotes``                    |``compactSize``                                 |The number of notes in the Issuance Action.                          |
 +-----------------------------+------------------------------+------------------------------------------------+---------------------------------------------------------------------+
-| 147 × ``nNotes``            |``vNotes``                    |``IssueNote[nNotes]``                           |A sequence of note descriptions within the Issuance Action.          |
+| 115 × ``nNotes``            |``vNotes``                    |``IssueNote[nNotes]``                           |A sequence of note descriptions within the Issuance Action.          |
```

Comment:
Independently, consider renaming `IssueNote` to `IssueNoteDescription`.

Reply:
Done.